frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Edit_HPA_Action.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Edit HPA action
+	As a user, I want to edit the HPA assigned to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment 
+    And user has added CPU resource limit
+    And user has added Memory resource limit
+    And user is in topology
+
+@regression
+Scenario: Edit HPA action form view
+   Given user has a workload with HPA assigned to it
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Edit Horizontal Pod Autoscaler option 
+   And user sees Edit Horizontal Pod Autoscaler form opens
+   And user sees values are prefilled
+   And user checks the name value but can not edit it
+   And user checks and edit the value for maximum and minimum pods
```

Comment:
Divide it in two statements

Reply:
You don't need to mention "checks". You can keep it as follows: 

```suggestion
   And user edits the value for maximum pods
   And user edits the value for minimum pods
```

Reply:
It will be check and edit as we need to check what is the existing value present in it